$NetBSD: patch-ap,v 1.2 2012/08/30 11:56:07 he Exp $

Do not embed rpath_blib into installable executables, as it will
refer to the build directory via rpath.

--- ./tools/dev/pbc_to_exe.winxed.orig	2012-06-26 01:18:19.000000000 +0000
+++ ./tools/dev/pbc_to_exe.winxed
@@ -617,7 +617,7 @@ function link_file(string objfile, strin
             config += 'parrot_config';
         else {
             config += 'install_config';
-            rpath  += $P0['rpath_lib'];
+            rpath  = $P0['rpath_lib'];
         }
     }
     else {
